src/providers/utils.ts

```diff
@@ -0,0 +1,91 @@
+// The async/queue library has a task-based interface for building a concurrent queue.
+
+import { providers } from "ethers";
+import lodash from "lodash";
+import { compareArrayResultsWithIgnoredKeys, compareResultsAndFilterIgnoredKeys } from "../utils/ObjectUtils";
+
+export { lodash };
+
+/**
+ * A record of error codes that correspond to fields that should be ignored when comparing RPC results.
+ * This is used to compare results from different providers that may have different, but still valid, results.
+ */
+const IGNORED_ERROR_CODES = {
```

Comment:
These aren't error codes, they are fields where we ignore their values in comparison, so I think we should update the name to something more like IGNORED_FIELDS or similar.

It's a good idea to pull these out, but I would've preferred to see this type of refactoring in a follow-up PR so it's easy to tell what changed.

Reply:
Pulled out and placed in: https://github.com/across-protocol/sdk/pull/695